manager/src/main/java/com/redhat/service/smartevents/processingerrors/api/ProcessingErrorsAPI.java

```diff
@@ -0,0 +1,76 @@
+import com.redhat.service.smartevents.infra.auth.IdentityResolver;
+import com.redhat.service.smartevents.infra.models.ListResult;
+import com.redhat.service.smartevents.infra.models.QueryResourceInfo;
+import com.redhat.service.smartevents.processingerrors.ProcessingErrorService;
+import com.redhat.service.smartevents.processingerrors.api.models.ProcessingErrorListResponse;
+import com.redhat.service.smartevents.processingerrors.models.ProcessingError;
+
+import io.quarkus.security.Authenticated;
+
+@Tag(name = "Processing Errors", description = "The API that allow the user to retrieve processing errors of a specific bridge instance.")
+@SecuritySchemes(value = {
+        @SecurityScheme(securitySchemeName = "bearer",
+                type = SecuritySchemeType.HTTP,
+                scheme = "Bearer")
+})
+@SecurityRequirement(name = "bearer")
+@Path(APIConstants.USER_API_BASE_PATH)
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+@Authenticated
+public class ProcessingErrorsAPI {
+
+    @Inject
+    IdentityResolver identityResolver;
+
+    @Inject
+    ProcessingErrorService processingErrorService;
+
+    @Inject
+    JsonWebToken jwt;
+
+    @APIResponses(value = {
+            @APIResponse(description = "Success.", responseCode = "200",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ProcessingErrorListResponse.class))),
+            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorsResponse.class))),
+            @APIResponse(description = "Unauthorized.", responseCode = "401", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorsResponse.class))),
+            @APIResponse(description = "Forbidden.", responseCode = "403", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorsResponse.class))),
+            @APIResponse(description = "Not found.", responseCode = "404", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorsResponse.class))),
+            @APIResponse(description = "Internal error.", responseCode = "500", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorsResponse.class)))
+    })
+    @Operation(summary = "Get the list of errors for a particular Bridge instance", description = "Get the list of errors for a particular Bridge instance for the authenticated user.")
+    @GET
+    @Path("{bridgeId}/errors")
+    public Response getBridgeErrors(@PathParam("bridgeId") String bridgeId, @Valid @BeanParam QueryResourceInfo queryInfo) {
+        ListResult<ProcessingError> errors = processingErrorService.getProcessingErrors(bridgeId, identityResolver.resolve(jwt), queryInfo);
```

Comment:
@kostola Personally, I think we should return `400` bad request on this endpoint if the user hasn't configured the "endpoint" error strategy otherwise a user of the API has no way of knowing that their not receiving processing errors because they forgot to configure the error handler.

Reply:
I already implemented this inside the `getProcessingError` method :+1:

Reply:
Ah yes. My mistake. Looks good.